src/Akka.Persistence.Sql.Linq2Db/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -140,6 +140,13 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                 .Member(r => r.Timestamp)
                 .HasColumnName(columnNames.Created);
 
+            //We can skip writing tags the old way by ignoring the column in mapping.
+            journalRowBuilder.Member(r => r.TagArr).IsNotColumn();
+            if (tableConfig.TagWriteMode == TagWriteMode.TagTable)
```

Comment:
Based on @to11mtm 's suggestion, we're going to need to be able to support multiple tag write modes. Can we address this in a subsequent PR?